components/private_channel/rust/ffi/src/private_channel.cpp

```diff
@@ -0,0 +1,78 @@
+/* Copyright 2020 The Brave Authors. All rights reserved.
```

Comment:
we need to end this .cpp .hpp convention that has been used in other ffi code

Reply:
- [ ] rename from `*.cpp` -> `*.cc` and `*.hpp` -> `*.h` across the FFI